packages/web3-eth-iban/src/index.ts

```diff
@@ -0,0 +1,162 @@
+import { toChecksumAddress, isAddress, leftPad, hexToNumber } from 'web3-utils';
+import { IbanLengthError } from 'web3-common';
+import { IbanOptions } from './types';
+
+/**
+ * Prepare an IBAN for mod 97 computation by moving the first 4 chars to the end and transforming the letters to
+ * numbers (A = 10, B = 11, ..., Z = 35), as specified in ISO13616.
+ */
+const _iso13616Prepare = (iban: string): string => {
+	const A = 'A'.charCodeAt(0);
+	const Z = 'Z'.charCodeAt(0);
+
+	const upperIban = iban.toUpperCase();
+	const modifiedIban = `${upperIban.substr(4)}${upperIban.substr(0, 4)}`;
```

Comment:
I think more good to use  `slice()` instead of `substr()` , as [substr](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) is not recommended